Source/Login/Backend/configuration/viper/identitites_configuration.go

```diff
@@ -16,3 +20,12 @@ func (c *identitiesConfiguration) Cookie() string {
 	}
 	return defaultIdentitiesCurrentUserCookieNameValue
 }
+
+func (c *identitiesConfiguration) TenantNamesMap() map[tenants.TenantID]string {
+	tenantNameMapRaw := viper.GetStringMapString(identitiesTenantNameMapKey)
+	tenantNameMap := map[tenants.TenantID]string{}
+	for tenantIDString, tenantName := range tenantNameMapRaw {
+		tenantNameMap[tenants.TenantID(tenantIDString)] = tenantName
+	}
+	return tenantNameMap
+}
```

Comment:
Also changing the name here, and some suggestions for possibly easier to understand code - do as you wish 🙂 

```suggestion
func (c *identitiesConfiguration) TenantNames() map[tenants.TenantID]string {
	if !viper.IsSet(identitiesTenantNamesKey) {
		return nil
	}
	tenantNames := map[tenants.TenantID]string{}
	for id, name := range viper.GetStringMapString(identitiesTenantNamesKey) {
		tenantNames[tenants.TenantID(id)] = name
	}
	return tenantNames
}
```

Reply:
Nice